agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/ConfigurationBuilder.java

```diff
@@ -173,6 +173,13 @@ private static Configuration loadConfigurationFile(Path agentJarPath) throws IOE
             }
         }
 
+        if (DiagnosticsHelper.isRpIntegration()) {
+            // users do not have write access to agent directory in rp integrations
```

Comment:
are we going to make this comment public on azure docs?

Reply:
i'm not sure about this specific comment, but we definitely need RP-specific azure docs on how to configure the agent